src/app/util/util.c

```diff
@@ -1400,3 +1400,19 @@ uint8_t emberAfMake8bitEncodedChanPg(uint8_t page, uint8_t channel)
         return channel | ENCODED_8BIT_CHANPG_PAGE_MASK_PAGE_0;
     }
 }
+
+EmberStatus emberAfScheduleServerTick(uint8_t endpoint, EmberAfClusterId clusterId, uint32_t delayMs)
+{
+    return EMBER_SUCCESS; // Stub for now.
```

Comment:
Please file issues to un-stub and link to them here and in the other stubs?

Reply:
this is covered by an issue already assigned to me